src/main/target/common_post.h

```diff
@@ -312,37 +314,55 @@
 #endif
 
 #ifdef USE_FLASH
+#if !defined(USE_FLASH_TOOLS)
 #define USE_FLASH_TOOLS
+#endif
+#if !defined(USE_FLASHFS)
 #define USE_FLASHFS
 #endif
+#endif
 
 #if (defined(USE_FLASH_W25M512) || defined(USE_FLASH_W25Q128FV)) && !defined(USE_FLASH_M25P16)
+#if !defined(USE_FLASH_M25P16)
```

Comment:
Maybe amend line above

Reply:
That was my first approach, but starts to get messy.

It's not code that's compiled so verbosity doesn't hurt.